apps/civil/civil-service/demo.yaml

```diff
@@ -7,7 +7,8 @@ spec:
   values:
     java:
       ingressClass: traefik-private
-      image: hmctspublic.azurecr.io/civil/service:pr-2516-9e851c8-20230324154959 #{"$imagepolicy": "flux-system:demo-civil-service"}
+      disableIngressClassAnnotation: true
+      image: hmctspublic.azurecr.io/civil/service:pr-2494-10c087a-20230328163346 #{"$imagepolicy": "flux-system:demo-civil-service"}
```

Comment:
same here for the latest has value of the image . . was this disableIngressClassAnnotation: true added by accident ?